src/ResourceManagement/Batch/Microsoft.Azure.Management.Batch/project.json

```diff
@@ -21,31 +21,36 @@
   },
 
   "dependencies": {
-    "Microsoft.Rest.ClientRuntime.Azure": "[3.3.1, 4.0)"
```

Comment:
since I have no idea when this PR will be merged and published I do not want to touch that part